lib/utils/read-user-info.js

```diff
@@ -9,14 +9,16 @@ exports.password = readPassword
 exports.username = readUsername
 exports.email = readEmail
 
-function read (opts) {
-  return Promise.resolve().then(() => {
-    log.clearProgress()
-    return readAsync(opts)
-  }).then(() => log.showProgress(), er => {
+async function read (opts) {
+  log.clearProgress()
+  try {
+    const input = await readAsync(opts)
+    log.showProgress()
+    return input
+  } catch (er) {
```

Comment:
```suggestion
    return input
  } finally {
    log.showProgress()
  }
```
and then you don't need to remember to re-throw the error.

Reply:
Avoiding catch and re-throw also maintains the original error throw callsite, which is helpful in debugging.